source/common/network/address_impl.cc

```diff
@@ -345,15 +345,19 @@ PipeInstance::PipeInstance(const sockaddr_un* address, socklen_t ss_len)
   }
   address_ = *address;
   if (abstract_namespace_) {
+    if (mode != 0) {
+      throw EnvoyException("Cannot set mode for Abstract AF_UNIX sockets");
```

Comment:
I think you are missing coverage for this error. Can you check the coverage report?

Reply:
Not sure why the [coverage report](https://306413-65214191-gh.circle-artifacts.com/0/coverage/coverage/proc/self/cwd/source/common/network/address_impl.cc.html#L349) says it's not covered but [this test covers it](https://github.com/envoyproxy/envoy/pull/8423/files/342c394125878a4e713dfb34024d8d192cd4ad20#diff-a97b9874a5d6e4ba7ea09da95a425b58R361-R367)

Reply:
You have 2 different places where you throw this exception. Are you sure you cover both? I would put an ASSERT here and make sure you are hitting both?

/wait

Reply:
[fixed](https://307506-65214191-gh.circle-artifacts.com/0/coverage/coverage/proc/self/cwd/source/common/network/address_impl.cc.html#L335)